legend-pure-runtime/legend-pure-runtime-java-engine-compiled/src/main/resources/org/finos/legend/pure/runtime/java/compiled/generation/processors/support/core/CoreGen.java

```diff
@@ -191,6 +192,42 @@ public static String toRepresentation(Object any, ExecutionSupport es)
         return "<" + Pure.manageId(any) + " instanceOf " + Pure.elementToPath((PackageableElement) CoreGen.safeGetGenericType(any, es)._rawType(), "::") + ">";
     }
 
+    public static RichIterable<? extends Root_meta_pure_functions_lang_KeyValue> processKeyExpressions(java.lang.Class<?> _class, Object instance, RichIterable<? extends Root_meta_pure_functions_lang_KeyValue> keyExpressions, ExecutionSupport es)
+    {
+        try
+        {
+            Method method = _class.getMethod("defaultValues", ExecutionSupport.class);
+            MutableList<org.eclipse.collections.api.tuple.Pair<String, RichIterable<? extends Object>>> vals = (MutableList<org.eclipse.collections.api.tuple.Pair<String, RichIterable<? extends Object>>>) method.invoke(instance, es);
```

Comment:
Can't you just cast _class to AbstractCompiledCoreInstance and then directly call defaultValues, rather than using reflection?